### PR TITLE
Add issiubclass

### PR DESCRIPTION
**Pull request by windelbouwman**

### CODE REVIEW

File: parser/src/lexer.rs [hunk 1]
```diff
@@ -20,8 +20,8 @@ pub enum LexicalError {
 
 #[derive(Clone, Debug, Default, PartialEq)]
 pub struct Location {
-    row: usize,
-    column: usize,
+    pub row: usize,
```
Comment (OddBloke) [inline parser/src/lexer.rs]: It looks like we only need these to be readable; would creating accessors be preferable?

Comment (windelbouwman) [reply parser/src/lexer.rs]: Valid point!

File: vm/src/builtins.rs [hunk 2]
```diff
@@ -221,7 +221,17 @@ fn builtin_isinstance(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     Ok(vm.context().new_bool(isinstance))
 }
 
-// builtin_issubclass
+fn builtin_issubclass(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    if args.args.len() != 2 {
+        panic!("isinstance(s) expects exactly two parameters");
```
Comment (cthulahoops) [inline vm/src/builtins.rs]: Should be issubclass?

File: vm/src/objtype.rs [hunk 3]
```diff
@@ -49,6 +49,11 @@ pub fn isinstance(obj: PyObjectRef, cls: PyObjectRef) -> bool {
     mro.into_iter().any(|c| c.is(&cls))
 }
 
+pub fn issubclass(typ: PyObjectRef, cls: PyObjectRef) -> bool {
+    let mro = _mro(typ).unwrap();
+    mro.into_iter().any(|c| c.is(&cls))
```
Comment (cthulahoops) [inline vm/src/objtype.rs]: Do you also need to check if typ itself is cls?

File: vm/src/builtins.rs (hunk 2 above)
Comment (windelbouwman) [reply vm/src/builtins.rs]: Yes, but i'll wait for the macro to be merged.

File: vm/src/objtype.rs (hunk 3 above)
Comment (windelbouwman) [reply vm/src/objtype.rs]: That's a very good question. I do not know. I thought of leaving this and facing the issue later :blush:

Comment (windelbouwman) [reply vm/src/objtype.rs]: Yes, but issubclass means that it is either of its own type, or of one of its inherited types.

**Pull request merged**